Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -5,6 +5,11 @@
 # common utilities
 ##############################################################################
 def _swapOfflineBSwithOnline(process):
+    import RecoVertex.BeamSpotProducer.onlineBeamSpotESProducer_cfi as _mod
+    process.BeamSpotESProducer = _mod.onlineBeamSpotESProducer.clone(
+        timeThreshold = cms.int32(999999) # for express allow >48h old payloads for replays. DO NOT CHANGE
```

Comment:
```suggestion
        timeThreshold = 999999 # for express allow >48h old payloads for replays. DO NOT CHANGE
```
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations. (similarly, for other files/cases).

Reply:
I was not sure it would have worked for parameters defined via `fillDescriptions`. Apparently it does